fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroupMgr.java

```diff
@@ -151,10 +155,30 @@ public void createResourceGroup(CreateResourceGroupStmt stmt) throws DdlExceptio
         LOG.info("Create resource group success: {}", resourceGroup);
     }
 
-    public void dropResourceGroup(DropResourceGroupStmt stmt) throws DdlException {
-        if (!Config.enable_resource_group) {
-            throw new DdlException("unsupported feature now, coming soon.");
+    public void alterResourceGroup(AlterResourceGroupStmt stmt) throws DdlException {
+        checkResourceGroupEnabled();
+
+        String resourceGroupName = stmt.getResourceGroupName();
+        Map<String, String> properties = stmt.getProperties();
+        ResourceGroup newResourceGroup;
+        writeLock();
+        try {
+            if (!nameToResourceGroup.containsKey(resourceGroupName)) {
+                throw new DdlException("Resource Group(" + resourceGroupName + ") does not exist.");
+            }
+            ResourceGroup resourceGroup = nameToResourceGroup.get(resourceGroupName);
+            newResourceGroup = ResourceGroup.create(resourceGroup, properties);
+            nameToResourceGroup.put(resourceGroupName, newResourceGroup);
```

Comment:
It seems that currently we not support partial attribute alter group.
Need a todo here/